app/src/test/java/org/simple/clinic/facility/change/FacilityChangeScreenControllerTest.kt

```diff
@@ -273,4 +273,65 @@ class FacilityChangeScreenControllerTest {
   fun `params for permission denials`(): List<RuntimePermissionResult> {
     return listOf(DENIED, NEVER_ASK_AGAIN)
   }
+
+  @Test
+  @Parameters("6", "7", "8")
```

Comment:
Can you also add `"5"` as a param here?
Just to make the test watertight

Reply:
The test will fail for `5`. The only values that can be used are > threshold (5)

Reply:
It passes for `"5"`.
I tested this.

Reply:
Whoops, you're right. The timer expires right after 5s. I'll update this.